created form for assigning student

kasech-isaac opened the pull request.

Jahteo commented on src/state/actions/index.js:
```diff
@@ -31,8 +38,23 @@ export const editVillage = (id, data) => () => {
+    .catch(err => console.dir(err));
+};
+
 export const fetchSchool = id => dispatch => {
```
Is this a duplicate from above?

yirano replied:
Nope! This one fetches the individual school and the other fetches all the school.

Jahteo reviewed (approved):
Due to time & branch conflict with feature/admin-dashboard, review was done over zoom. We checked full UI functionality of both this branches, fixed problems that had arisen and pushed. Comments on code should still be added later

kasech-isaac committed:
created form for assigning student

kasech-isaac committed:
Assign Student

https://trello.com/c/jQFt0LOR/13-as-a-headmaster-i-want-to-be-able-to-add-students-to-the-mentors-schedule-so-that-we-have-students-assigned-to-a-mentor

markanator committed:
Merge branch 'main' into Feature/assign_student

markanator reviewed (changes requested):
Hey Kasi, I saw see there are some empty files in this PR. Can you remove those three files? Other than that, looks good!

kasech-isaac committed:
resolved merge conflicts

kasech-isaac committed:
Merge branch 'Feature/assign_student' of https://github.com/Lambda-School-Labs/village-book-builders-fe-b into Feature/assign_student

kasech-isaac committed:
student assign tomentee

changed library select to the right  .map()   name.

markanator commented on src/components/pages/Headmaster/Signup.js:
```diff
@@ -0,0 +1,373 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import moment from 'moment';
+import MentorandMenteeR from './MentorandMenteeR';
```
Kasi, what does this file do?

kasech-isaac committed:
icon key description

created icon key description and deleted "MentorandMenteeR.js" file

kasech-isaac committed:
assign-student

deleted Contact input.

markanator committed:
remove hardcoded api calls

josesustaita committed:
Merge branch 'main' into Feature/assign_student

josesustaita committed:
Merge branch 'main' into Feature/assign_student

markanator committed:
Merge branch 'main' into Feature/assign_student

markanator committed:
Merge branch 'main' of https://github.com/Lambda-School-Labs/village-book-builders-fe-b into main

markanator committed:
Merge branch 'main' into Feature/assign_student

markanator committed:
connected modal with calendar and redux

josesustaita reviewed (approved):
Nice! We now have a form to assign a student/mentor to a session
